consensus/src/quorum_store/batch_proof_queue.rs

```diff
@@ -515,10 +532,27 @@ impl BatchProofQueue {
             }
         }
 
+        let max_batch_ts_usecs = min_batch_age_usecs
```

Comment:
nit: the name is not very clear to me, probably should be max_batch_creation_usecs